src/api/app/controllers/statistics/maintenance_statistics_controller.rb

```diff
@@ -1,12 +1,24 @@
 module Statistics
   class MaintenanceStatisticsController < ApplicationController
-    skip_before_action :extract_user
+    skip_before_action :extract_user, :require_login
 
     def index
       @project = Project.get_by_name(params[:project])
-      @maintenance_statistics = MaintenanceStatisticDecorator.wrap(
-        MaintenanceStatistic.find_by_project(@project)
-      )
+      if @project.is_a?(String)
+        remote_instance, remote_project = Project.find_remote_project(@project)
+        # Port get's explicitly set to properly redirect from dev machines (port 3000)
+        # to instances running in production mode.
+        redirect_to maintenance_statistics_url(
```

Comment:
> hennevogel 2 hours ago
> Why don't you fetch the data from remote and render it?

I don't see the point of doing additional work on the local instance. It's not like the local host would have to process the data in any way. We pass them through anyway, so why not redirecting the request?